src/toil_vg/vg_call.py

```diff
@@ -552,18 +524,62 @@ def run_calling(job, context, xg_file_id, alignment_file_id, alignment_index_id,
         clipped_chunk_offset = chunk_i * context.config.call_chunk_size - chunk_i * context.config.overlap
         cur_path_offset[chunk_bed_chrom] += 1
 
-        call_job =  child_job.addChildJobFn(run_call_chunk, context, chunk_bed_chrom, chunk_i,
-                                            len(bed_lines),
-                                            chunk_bed_start, clipped_chunk_offset,
-                                            None, vg_chunk_file_id, gam_chunk_file_id,
-                                            path_size[chunk_bed_chrom], offset_map[chunk_bed_chrom],
-                                            sample_name, genotype, recall,
-                                            cores=context.config.misc_cores,
-                                            memory=context.config.misc_mem, disk=context.config.misc_disk)
-        clip_file_ids.append(call_job.rv(0))
-        call_timers.append(call_job.rv(1))
+        chunk_info = {
+            'chrom' : chunk_bed_chrom,
+            'chunk_i' : chunk_i,
+            'chunk_n' : chunk_counts[chunk_bed_chrom],
```

Comment:
@cmarkello replacing len(bed_lines) with chunk_counts[chunk_bed_chrom] here fixes a bug where the last overlap/2 bases of all but the last chromosome get clipped out when the input gam wasn't split by chromosomes.  Mentioning because it may affect your wdl script.